slotfills/sections/fieldRouter.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { v4 as uuidv4 } from 'uuid';
+
+import TextField from './text-field';
+import TextareaField from './textarea-field';
+import RichtextField from './richtext-field';
+
+const FieldRouter = ({
+  field,
+  field: {
+    field_class: fieldClass,
+    attributes: {
+      rows = null,
+    } = {},
+    label = '',
+  },
+  index,
+  valueHook,
+}) => {
+  const fieldKey = uuidv4();
+  return (
+    <>
+      {fieldClass === 'text' && rows === null ? (
```

Comment:
This could be refactored to use the early return pattern, which I think would result in cleaner code. For example:

```jsx
if (fieldClass === 'text') {
  if (rows) {
    return <TextareaField .../>;
  }
  return <TextField .../>;
} else if (fieldClass === 'richtext') {
  return <RichtextField .../>;
}
```